res/translators/csharp/SplashKit/classes/resource_classes.cs.erb

```diff
@@ -3,15 +3,23 @@
 %>
 
 <%
-  @classes.select{ |k, c| ! c[:is_struct] }.each do | class_id, the_class |
+  @classes.select { |k, c| !c[:is_struct] }.each do |class_id, the_class|
     class_name = class_id.to_s.to_pascal_case
 
     if the_class[:is_alias]
 %>
+/// <summary>
+/// This class represents <%= class_name %>, which wraps a pointer to SplashKit resources.
+/// </summary>
 public class <%= class_name %> : PointerWrapper
 {
   private <%= class_name %>(IntPtr ptr) : base(ptr, true) {}
 
+  /// <summary>
+  /// Fetches or creates an instance of <%= class_name %> from the given pointer.
+  /// </summary>
+  /// <param name="ptr">The pointer to the SplashKit resource.</param>
+  /// <returns>An instance of <%= class_name %>.</returns>
```

Comment:
Okay, so we should remove this as this is an internal function and our users can never call this, and so can never access this information.